### ISSUE TITLE
NTR peptidoglycan hydrolase activity

### ISSUE THREAD

**Issue by jimhu-tamu** [geneontology/go-ontology#12436]
This would be an umbrella term for the different kinds of enzymes that cleave peptidoglycan. 

This [chapter](http://www.oecd.org/tad/crp/Endolysins1.pdf) on phage endolysins describes how enzymes that degrade peptidoglycan can be divided into subclasses. 

peptidoglycan hydrolase activity is_a GO:0016787 ! hydrolase activity.

This would become an is_a parent for each of the 7 different types of activities shown in fig 1 of the linked chapter. Some of these are existing terms:

OTRs for children of the new term

- GO:0003796 ! lysozyme activity
- GO:0008933 ! lytic transglycosylase activity
- GO:0008745 ! N-acetylmuramoyl-L-alanine amidase activity

NTRs
- peptidoglycan N-acetylglucosaminidase activity is_a GO:0016810 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds
- peptidoglycan endopeptidase activity
 - peptidoglycan stem peptide endopeptidase activity
 - peptidoglycan cross-bridge peptide endopeptidase activity (synonym: lysostaphin activity NARROW)

**Comment by jimhu-tamu** [comment]
After discussing this item with my local expert, I have some changes for this item. I think we still need an umbrella term for an activity that degrades peptidoglycan, but he points out that the lytic transglycosylases are not hydrolases, so although the review calls them peptidoglycan hydrolases, we should use that as a synonym, not a term name. This also means that the umbrella term should not be a child of GO:0016787 ! hydrolase activity, but rather a child of GO:0003824 ! catalytic activity, unless a better intermediate parent could be proposed.

As for the name for the new term, he proposes muralytic activity. The problem with this is that while it has a history of being used in the literature for anything that degrades bacterial cell walls, current [dictionaries](https://en.wiktionary.org/wiki/muralytic) give it as a more specific activity that removes muramyl groups. More later, I hope...

**Comment by ukemi** [comment]
Hi Jim,

Let me know when you get an update on this. We do have the process peptidoglycan catabolic process.  If these activities are always used in the process of degrading peptidoglycans, we could go the route of making function-process links between them and the process term. It wouldn't be as specific but it might give you the grouping you want.

-D

**Comment by jimhu-tamu** [comment]
Talking to Ry Young, he argues that the online dictionaries are wrong, and that we should go with muralytic activity. I'm not sure about peptidoglycan catabollic process, since it's not used for energy.

**Comment by ukemi** [comment]
Hi Jim,

Sorry about taking so long on this, but I just saw that you had added a new comment some time ago. What about a definition and a reference?

muralytic activity- A catalytic activity that contributes to the degradation of peptidoglycan?

This creates a dilemma with respect to the proposed children. According to the definition of lysozyme, it can act on molecules other than peptidoglycan, so it would not necessarily be a correct child. I guess I could create a subtype that specifically targets peptidoglycan. It puts us in kind of a difficult spot. Are the other enzymes you mention above specific to peptidoglycan?

-D

**Comment by jimhu-tamu** [comment]
Hi David, 

No problem, I'm ridiculously behind on lots of stuff too. I guess it depends on what you mean by specific to peptidoglycan. I was not aware of what you are saying about lysozymes, in that I thought that they were defined as having bacterial cell walls (peptidoglycan) as their biological substrate.

I'm wondering if this is analogous to proteases/peptideases. There are artificial peptide substrates that are used to measure activity, but we still think of polypeptides as the substrate.

**Comment by ukemi** [comment]
Hi JIm,

Here is the def of lysozyme in GO (and EC):
Catalysis of the hydrolysis of the beta-(1->4) linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan and between N-acetyl-D-glucosamine residues in chitodextrins.

It is the last part that has me wondering. Could it also be a chitinase. Is it naturally? If they only have bacterial cell walls as their biological substrate, then we are good and maybe we should get rid of that last clause.

-D

**Comment by jimhu-tamu** [comment]
Didn't know about the chitodextrin (sp?) part. Searching pubmed for chitodextrin gives one paper in Russion based on chitodextran. Searching for chitobiose AND lysozyme gives a bunch of papers as lysozyme inhibitors. 

I am not finding chitodextrin in ChEBI.

**Comment by ukemi** [comment]
If you look at the EC entry for chitinase, http://enzyme.expasy.org/EC/3.2.1.14, it says some of them display the activity described for lysozyme. 

But you are right, a pubmed search just gives that one paper. 

Searching the pubmed for lysozyme AND chitinase gives papers like this: PMID25135943.
 
It seems to me that the definition that we have for lysozyme is based on the bifunctional activity of some of these enzyme molecules. It seems that some enzymes called lysozyme carry out lysozyme activity which should properly be defined as:
Catalysis of the hydrolysis of the beta-(1->4) linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan

and chitinase activity: Catalysis of the hydrolysis of (1->4)-beta linkages of N-acetyl-D-glucosamine (GlcNAc) polymers of chitin and chitodextrins.

Do you agree? How can something that seemed so simple on the surface end up taking me an afternoon to figure out?

**Comment by jimhu-tamu** [comment]
Sounds good to me. And isn't it always like that? I'm prepping some examples for CACAO training tonight and the examples I'm finding don't make sense (not for this term).

**Comment by ukemi** [comment]
Hi Jim,

The diff is below. Have a look and if you spot any concerns, give me a shout. I made a couple of changes. I named the new term peptidoglycan muralytic activity to distinguish it from enzymes that degrade plant cell walls. apparently they are also muralytic. I also made the N-acetylglucosaminidase a type of hexosaminidase. That seemed appropriate.

 id: GO:0003796
 name: lysozyme activity
 namespace: molecular_function
-def: "Catalysis of the hydrolysis of the beta-(1->4) linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan and between N-acetyl-D-glucosamine residues in chitodextrins." [EC:3.2.1.17]
+def: "Catalysis of the hydrolysis of the beta-(1->4) linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan." [EC:3.2.1.17, PMID:22748813]
 subset: gosubset_prok
 synonym: "1,4-N-acetylmuramidase activity" EXACT []
 synonym: "globulin G" RELATED [EC:3.2.1.17]
@@ -33562,6 +33562,7 @@
 xref: EC:3.2.1.17
 xref: MetaCyc:3.2.1.17-RXN
 is_a: GO:0004553 ! hydrolase activity, hydrolyzing O-glycosyl compounds
+is_a: GO:0061783 ! peptidoglycan muralytic activity
 
 [Term]
 id: GO:0003797
@@ -105314,7 +105315,7 @@
 id: GO:0008745
 name: N-acetylmuramoyl-L-alanine amidase activity
 namespace: molecular_function
-def: "Catalysis of the hydrolysis of the link between N-acetylmuramoyl residues and L-amino acid residues in certain bacterial cell-wall glycopeptides." [EC:3.5.1.28]
+def: "Catalysis of the hydrolysis of the link between N-acetylmuramoyl residues and L-amino acid residues in certain bacterial cell-wall glycopeptides." [EC:3.5.1.28, PMID:22748813]
 subset: gosubset_prok
 synonym: "acetylmuramoyl-alanine amidase activity" EXACT [EC:3.5.1.28]
 synonym: "acetylmuramyl-alanine amidase activity" EXACT [EC:3.5.1.28]
@@ -105330,6 +105331,7 @@
 xref: EC:3.5.1.28
 xref: MetaCyc:3.5.1.28-RXN
 is_a: GO:0016811 ! hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides
+is_a: GO:0061783 ! peptidoglycan muralytic activity
 
 [Term]
 id: GO:0008746
@@ -108268,12 +108270,13 @@
 id: GO:0008933
 name: lytic transglycosylase activity
 namespace: molecular_function
-def: "Catalysis of the specific cleavage of the beta-(1->4) glycosidic linkage between N-acetylmuramyl and N-acetylglucosaminyl residues in peptidoglycan, with the concomitant formation of 1,6-anhydro-N-acetylmuramyl residues." [PMID:10964424]
+def: "Catalysis of the specific cleavage of the beta-(1->4) glycosidic linkage between N-acetylmuramyl and N-acetylglucosaminyl residues in peptidoglycan, with the concomitant formation of 1,6-anhydro-N-acetylmuramyl residues." [PMID:10964424, PMID:22748813]
 subset: gosubset_prok
 synonym: "murein transglycosylase B activity" NARROW []
 synonym: "peptidoglycan lytic transglycosylase activity" EXACT []
 xref: EC:3.2.1.-
 is_a: GO:0016757 ! transferase activity, transferring glycosyl groups
+is_a: GO:0061783 ! peptidoglycan muralytic activity
 
 [Term]
 id: GO:0008934
@@ -403549,6 +403552,54 @@
 creation_date: 2016-09-06T13:29:49Z
 
 [Term]
+id: GO:0061783
+name: peptidoglycan muralytic activity
+namespace: molecular_function
+def: "A catalytic activity that contributes to the degradation of peptidoglycan." [GOC:dph, GOC:jh, PMID:22748813]
+is_a: GO:0003824 ! catalytic activity
+created_by: dph
+creation_date: 2016-09-07T15:47:49Z
+
+[Term]
+id: GO:0061784
+name: peptidoglycan N-acetylglucosaminidase activity
+namespace: molecular_function
+def: "Catalysis of the hydrolysis of (1->4)-beta linkages of N-acetyl-D-glucosamine (GlcNAc) from peptidoglycan." [GOC:dph, GOC:jh, PMID:22748813]
+is_a: GO:0015929 ! hexosaminidase activity
+is_a: GO:0061783 ! peptidoglycan muralytic activity
+created_by: dph
+creation_date: 2016-09-07T16:17:03Z
+
+[Term]
+id: GO:0061785
+name: peptidoglycan endopeptidase activity
+namespace: molecular_function
+def: "An endopeptidase activity that uses peptidoglycan as a substrate." [GOC:dph, GOC:jh, PMID:22748813]
+is_a: GO:0004175 ! endopeptidase activity
+is_a: GO:0061783 ! peptidoglycan muralytic activity
+created_by: dph
+creation_date: 2016-09-07T16:30:48Z
+
+[Term]
+id: GO:0061786
+name: peptidoglycan stem peptide endopeptidase activity
+namespace: molecular_function
+def: "A peptidoglycan endopeptidase activity that acts on a stem peptide of peptidoglycan." [GOC:dph, GOC:jh, PMID:22748813]
+is_a: GO:0061785 ! peptidoglycan endopeptidase activity
+created_by: dph
+creation_date: 2016-09-07T16:47:04Z
+
+[Term]
+id: GO:0061787
+name: peptidoglycan cross-bridge peptide endopeptidase activity
+namespace: molecular_function
+def: "A peptidoglycan endopeptidase activity that acts on a peptidoglycan cross-bridge." [GOC:dph, GOC:jh, PMID:22748813]
+synonym: "lysostaphin activity" NARROW [GOC:dph, GOC:jh, PMID:22748813]
+is_a: GO:0061785 ! peptidoglycan endopeptidase activity
+created_by: dph
+creation_date: 2016-09-07T16:54:02Z
